### PR TITLE
merge Part 2a and 2b to main branch

### PR DESCRIPTION
**Pull request by ruoxiran**

### CODE REVIEW

File: content-usable/design.html [hunk 1]
```diff
@@ -25,342 +24,335 @@ <h2>
...
+      	Use familiar design patterns, terms and icons to help users who struggle to remember new designs. Ensure the look, location and interaction of controls and other elements are familiar and consistent across the site. 
+      </p>
+      <p>
+      	Show a clear relationship between controls and the content they effect to help users understand the effect of possible actions and reduce potential confusion.
       </p>
+  
     
       <section> 
         <h3>Pattern: Make the Purpose of Your Page Clear</h3>
           <section><h5 class="coga-5">User Need</h5>
```
Comment (SteveALee) [inline content-usable/design.html:56]: @ruoxiran  Aren't we missing h4? I know we need to keep avoid the lower headings as they are not clearly formatted. Or was that h4 that was bad so we skip it?

Comment (ruoxiran) [reply content-usable/design.html:56]: When I merged, I did not modify the original heading, but this is not a problem. W3C repec.js will reset all headings and ids in HTML. If you inspect the html code of http://w3c.github.io/coga/content-usable/, you will find all headings and ids be reformatted automatically.

**Comment by ruoxiran** [comment]
part 3 has been merged.

**Review by rachaelbradley — APPROVED** [review]
I compared these and fixed any mistakes with my recent push.

**Pull request merged**